test_project/features/links/t_links/t_links.feature

```diff
@@ -1,7 +1,7 @@
 @fixture.set_workdir
```

Comment:
Please change this file back to how it was and only use the t_links_bigquery.feature file.